frontend/taipy-gui/src/components/Taipy/Metric.tsx

```diff
@@ -133,3 +166,63 @@ const Metric = (props: MetricProps) => {
 }
 
 export default Metric;
+
+const darkTemplate = {
```

Comment:
Is this a copy of what's in chart.tsx ?
If so, it should be put in a common ts file.

Reply:
I've refactored in the latest commit

Reply:
👍